api/src/config.ts

```diff
@@ -14,34 +18,32 @@ export const SECURE_PORT = process.env.SECURE_PORT || 4096;
 export const INSECURE_PORT = process.env.INSECURE_PORT || 3001;
 
 // Keys for SSL
-export const KEY_PATH = process.env.KEY_PATH || 'server.key'
-export const CERT_PATH = process.env.CERT_PATH || 'server.cert'
+export const KEY_PATH = getEnv('KEY_PATH');
+export const CERT_PATH = getEnv('CERT_PATH');
 
 // MySQL config for students database
 export const MYSQL_STUDENTS_CONFIG = {
-  host: process.env.MYSQL_HOST || die('mysql host'),
-  port: parseInt(process.env.MYSQL_PORT || die('mysql port'), 10),
-  user: process.env.MYSQL_USER || die('mysql username'),
-  password: process.env.MYSQL_PASSWORD || die('mysql password'),
-  database: process.env.STUDENTS_DB || die('students db'),
+  host: getEnv('MYSQL_HOST'),
+  port: parseInt(getEnv('MYSQL_PORT'), 10),
+  user: getEnv('MYSQL_USER'),
+  password: getEnv('MYSQL_PASSWORD'),
+  database: getEnv('STUDENTS_DB'),
 };
 
 // MySQL config for courses database (just worksheets)
 export const MYSQL_COURSES_CONFIG = {
-  host: process.env.MYSQL_HOST || die('mysql host'),
-  port: parseInt(process.env.MYSQL_PORT || die('mysql port'), 10),
-  user: process.env.MYSQL_USER || die('mysql username'),
-  password: process.env.MYSQL_PASSWORD || die('mysql password'),
-  database: process.env.COURSES_DB || die('courses db'),
+  host: getEnv('MYSQL_HOST'),
+  port: parseInt(getEnv('MYSQL_PORT'), 10),
```

Comment:
Do you really need to specify base 10?

Reply:
Yeah, I think it's an eslint autofix: https://eslint.org/docs/rules/radix

Reply:
Seems unnecessary, but ok.